packages/web-app-files/src/components/Search/Preview.vue

```diff
@@ -70,12 +71,19 @@ export default {
       }
 
       return this.$gettext('Personal')
+    },
+    displayThumbnails() {
+      return !this.configuration.options.disablePreviews
```

Comment:
Needs optional chaining, `configuration.options` does not necessarily exist. Could you do that in a followup PR?